dir/path.go

```diff
@@ -1,88 +1,71 @@
 package dir
```

Comment:
It is better to document the package. like
```golang
// Package dir does bla bla bla...
package dir
```

Reply:
Added doc.